x-pack/plugins/siem/public/resolver/view/index.tsx

```diff
@@ -33,6 +33,10 @@ const StyledResolverContainer = styled.div`
   display: flex;
   flex-grow: 1;
   contain: layout;
+  cursor: grab;
+  &:active {
+    cursor: grabbing;
+  }
```

Comment:
Test subject indicated that it wasn't obvious that he could grab & move the scene, so this should help.